apps/android/boot.js

```diff
@@ -268,7 +295,7 @@
   // Health tracking
   Bangle.on('health', health=>{
     if (actInterval===undefined) // if 'realtime' we do it differently
-      gbSend({ t: "act", stp: health.steps, hrm: health.bpm });
+      gbSend({ t: "act", stp: health.steps, hrm: health.bpm, rt:1 });
```

Comment:
Pretty sure this one *isn't* realtime? It only happens every 10 minutes when we get health data - so this is basically the health data that we're trying to sync above and we need to not have `rt` set